app/services/imports/scheme_location_import_service.rb

```diff
@@ -26,6 +26,10 @@ def create_scheme_location(xml_document)
       4 => "(Registered nursing care home)",
     }.freeze
 
+    LA_NAME_TO_CODE = {
```

Comment:
So this will be a list of maps when you have that data is that right?

Reply:
Yup, all there now